dev/io.openliberty.microprofile.metrics.internal.5.0/bnd.bnd

```diff
@@ -0,0 +1,76 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at 	
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0.${libertyBundleMicroVersion}
+
+-sub: *.bnd
+
+IBM-Authorization-Roles: com.ibm.ws.management
+IBM-Web-Extension-Processing-Disabled: true
+
+Import-Package: \
+	javax.enterprise.inject; version="[1.1,3)",\
+	javax.enterprise.util; version="[1.1,3)",\
+	javax.annotation; version=!,\
+	javax.interceptor; version=!,\
+	javax.enterprise.context.spi; version=!,\
+	org.eclipse.microprofile.config; version="[3.0,4)",\
+	*
+
+Private-Package: \
+	com.ibm.ws.microprofile.metrics.resources,\
+	com.ibm.ws.microprofile.metrics.exceptions
+
+Export-Package: \
+	io.openliberty.microprofile.metrics50.internal,\
+	io.openliberty.microprofile.metrics50.internal.impl,\
+	io.openliberty.microprofile.metrics50.helper,\
+	io.openliberty.smallrye.metrics.adapters,\
+	io.openliberty.smallrye.metrics.cdi.adapters,\
+	io.openliberty.metrics
+
+-dsannotations-inherit: true
+-dsannotations: \
+	io.openliberty.microprofile.metrics50.internal.MetricsConfig,\
+	io.openliberty.microprofile.metrics50.internal.impl.SharedMetricRegistries,\
+	io.openliberty.metrics.SmallryeMetricsCDIMetadata,\
+	io.openliberty.microprofile.metrics50.internal.ApplicationListener30
+
+WS-TraceGroup: METRICS
+
+-buildpath: \
+    org.eclipse.osgi;version=latest,\
+	com.ibm.websphere.org.osgi.core;version=latest,\
+	com.ibm.ws.app.manager.wab;version=latest,\
+	com.ibm.ws.container.service;version=latest,\
+	com.ibm.ws.adaptable.module;version=latest,\
+	com.ibm.ws.logging;version=latest,\
+	com.ibm.ws.rest.handler;version=latest,\
+	com.ibm.ws.classloading;version=latest,\
+	com.ibm.ws.webcontainer;version=latest,\
+	com.ibm.ws.container.service;version=latest,\
+	com.ibm.ws.container.service.compat;version=latest,\
+	com.ibm.ws.kernel.boot.core;version=latest,\
+	com.ibm.ws.kernel.service;version=latest,\
+	com.ibm.websphere.org.osgi.service.component;version=latest,\
+	com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
+	com.ibm.wsspi.org.osgi.service.event;version=latest,\
+	io.openliberty.org.eclipse.microprofile.metrics.5.0;version=latest,\
+	com.ibm.websphere.appserver.spi.kernel.service;version=latest,\
+	com.ibm.websphere.appserver.spi.httptransport;version=latest,\
+	com.ibm.websphere.javaee.servlet.3.1;version=latest,\
+	com.ibm.websphere.appserver.spi.ssl;version=latest,\
+	com.ibm.websphere.javaee.cdi.2.0;version=latest, \
+	com.ibm.ws.cdi.interfaces;version=latest,\
```

Comment:
These need to be the Jakarta EE 10 versions ... but if the PB passes like this then get it merged in and fix it in a follow up.